gno.land/pkg/gnoland/genesis.go

```diff
@@ -138,3 +141,20 @@ func LoadPackage(pkg gnomod.Pkg, creator bft.Address, fee std.Fee, deposit std.C
 
 	return tx, nil
 }
+
+func DefaultGenState() GnoGenesisState {
+	authGen := auth.DefaultGenesisState()
+	gp, err := std.ParseGasPrice(InitGasPrice)
```

Comment:
Can `InitGasPrice` be a `GasPrice` struct so it doesn't need to be parsed?

Reply:
InitGasPrice is a default genesis value as a constant. Go only allows primitive types as constants.

Reply:
This only needs to be parsed once when we start and stop a new node. The drawback of using a global variable is that it could be updated elsewhere in the code.